books/thirdparty/livelibru.py

```diff
@@ -102,11 +120,14 @@ def search_books_with_reviews(search_request_text):
     results = []
```

Comment:
Lol, I can't leave comment on lines above. But can you add type hint on search_books_with_reviews  that it returns list[Book] ?